### DISCUSSION TITLE
French lemmatizer lemmatizes differently `Expliquez` and `expliquez`

### DISCUSSION THREAD

**Discussion by alephpi** [explosion/spaCy#12535] [Help: Best practices]
Hi, I'm new to spaCy and I'm currently using `fr_core_news_sm` model to do some NLP job. I find that the lemmatizer (when calling `Token.lemma_`) doesn't treat `Expliquez` and `expliquez` evenly, where the first lemma is `expliquez` and the second lemma is `expliquer` (what I want indeed). I wonder 
1. why the lemmatizer has this discrimination?
2. what is the best practice to get the truly lemma (whose lemma is itself)? A quick and dirty way is of course to do `nlp(lemma)` repeatedly but is there a better practice?

**Comment by alephpi** [comment]
More observations:
`expliquez` is falsely pos-tagged as `ADP` rather than `VERB`

**Comment by alephpi** [comment]
Well, turns out that better use the large model `fr_core_news_lg`, that way all the issues are gone.

**Reply by svlandeg** [reply]
Thanks for reporting back! In general, the large ("lg") models will indeed be more accurate than the small ("sm") ones.
